src/todo-list.js

```diff
@@ -0,0 +1,180 @@
+/**
```

Comment:
Choose a code style and go with it.
You can either use semicolon or not use it - but not both.